tests/_test_core_strings.py

```diff
@@ -188,3 +193,168 @@
 
 }); // END REQUIRE.JS block
 </script>"""
+
+STANDALONE_HTML_STRING = """
```

Comment:
Got it, yeah, the reason why we are fine is because of this magic line:
https://github.com/biocore/emperor/blob/new-api/tests/_test_core_strings.py#L8

It asks flake8 to ignore all checks inside this file.

On (Jun-28-16|12:15), Antonio Gonzalez wrote:
> > @@ -188,3 +193,168 @@
> >  
> >  }); // END REQUIRE.JS block
> >  </script>"""
> > +
> > +STANDALONE_HTML_STRING = """
> 
> K, was wondering if it will be an issue with pep8 but it seems like we are fine.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/510/files/378f456b29428865360dc7c3ad7a0beaed9e08b2#r68823422